book/book.toml

```diff
@@ -0,0 +1,15 @@
+[book]
+title = "The Goose Book"
+authors = ["Jeremy Andrews"]
+description = "Load testing framework, inspired by Locust, implemented with Rust."
+language = "en"
+
+[rust]
+edition = "2018"
+
+[build]
+build-dir = "html"
+create-missing = true
```

Comment:
Do not override, leave the default: I'd prefer the build generates an error instead of creating empty pages.

Reply:
Resolving as can't fix in 3-6 months.